changelog/1.26.3.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "1.26.3"
+description: "Released on 03/11/2022"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.26.3.
+
+### Features ✨
+
+- infra: added debug level logging for Docker in CVM outer container.
+
+### Bug fixes 🐛
+
+- infra: fixed memory leak due to unclosed `time.Ticker`.
```

Comment:
This `time.Ticker` detail is not useful for customers. Instead, say something like `fixed memory leak in workspace create process`.